### PR TITLE
UX: Multichain: Send flow: Send page skeleton

### PR DESCRIPTION
**Pull request by darkwing**
## **Description**
This PR conditionally shows our new Send page based on `MULTICHAIN` environment variable.  No functional expectation here, simply basic layout.

## **Manual testing steps**

1.  Run `MULTICHAIN=1 yarn start`
2. Click into any token
3. Click the "Send" button
4. See this page.

## **Screenshots/Recordings**

SCR-20231008-quyx

## **Related issues**

Fixes https://github.com/MetaMask/MetaMask-planning/issues/1439

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained:
  - [ ] What problem this PR is solving.
  - [ ] How this problem was solved.
  - [ ] How reviewers can test my changes.
- [ ] I’ve indicated what issue this PR is linked to: Fixes #???
- [ ] I’ve included tests if applicable.
- [ ] I’ve documented any added code.
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/components/multichain/pages/send/send.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        }
+      >
+        {t('sendAToken')}
+      </Header>
+      <Content></Content>
+      <Footer>
+        <ButtonSecondary block>{t('cancel')}</ButtonSecondary>
+        <ButtonPrimary block disabled>
+          {t('confirm')}
+        </ButtonPrimary>
```
Comment (NidhiKJha) [inline ui/components/multichain/pages/send/send.js:33]: Nit: Both these buttons should be LG?

File: ui/components/multichain/pages/send/index.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.multichain-send-page {
+  width: 408px;
+}
```
Comment (NidhiKJha) [inline ui/components/multichain/pages/send/index.scss:3]: Should we include the height css in this skeleton as well?

**Review by NidhiKJha — COMMENTED** [review]
Looks great! Just left a couple of comments

File: ui/components/multichain/pages/send/send.js (hunk 1 above)
Comment (darkwing) [reply ui/components/multichain/pages/send/send.js]: Nice catch!

File: ui/components/multichain/pages/send/index.scss (hunk 2 above)
Comment (darkwing) [reply ui/components/multichain/pages/send/index.scss:3]: Nope, the height will stretch to what's available, since height will vary based on contents.

**Review by vthomas13 — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Pull request merged**